app/cmd/cli/root.go

```diff
@@ -60,18 +61,19 @@ func init() {
 	rootCmd.PersistentFlags().IntVar(&blockTime, "blockTime", 1, "how often should the network create blocks")
 	rootCmd.PersistentFlags().BoolVar(&testnet, "testnet", false, "would you like to connect to Pocket Network testnet")
 	rootCmd.PersistentFlags().BoolVar(&simulateRelay, "simulateRelay", false, "would you like to be able to test your relays")
+	rootCmd.Flags().BoolVar(&keybase, "keybase", true, "run wiith keybase, if disabled allows you to stake for the current validator only. providing a keybase is still neccesary for staking for apps & sending transactions")
 	rootCmd.AddCommand(startCmd)
 	rootCmd.AddCommand(resetCmd)
 	rootCmd.AddCommand(version)
 }
 
 // startCmd represents the start command
 var startCmd = &cobra.Command{
-	Use:   "start",
+	Use:   "start --keybase=<keybase>",
```

Comment:
Start without flag is not possible ?

Reply:
https://github.com/pokt-network/pocket-core/blob/3eb2fbb15ddeaf2d0fd3ed3a42f2f632e2ca1589/app/cmd/cli/root.go#L64

flag has a default value, unless a flag is mark as required cobra won't require it 

`rootCmd.MarkFlagRequired("keybase")` is the only way to make sure it's neccesary.